### DISCUSSION TITLE
Not getting accepted

### DISCUSSION THREAD

**Discussion by Iced-Coded** [community/community#45512] [GitHub Education] [answered]
Hello! I've sended around **4** applications, first of them was my school id, which, was denied due to it being in Ukrainian language, then, I've sended a screenshot of the school's online diary, which, proves that I'm a student, but, it has my First, Middle and Last name! I wouldn't like to disclose my Middle name, but, other applications were declined due to "not having enough info on the GitHub profile".
![image](https://user-images.githubusercontent.com/48327170/215058310-8f1ecbfd-a51a-4de7-b7e2-30320e5f16f9.png)

**Comment by jge162** [comment, marked as answer]
Just update your profile with as much info as you can, name, etc. and submit transcripts with your name matching and use your school email. I was approved that way.

**Reply by jge162** [reply]
just complete your github user profile. it says it in the email.

**Reply by jge162** [reply]
then reapply

**Reply by Iced-Coded** [reply]
I did enter my full name on the profile, and made a description. Also, our school don't have school emails :(

**Reply by jge162** [reply]
hm, is it a college? do you have transcripts?

**Reply by Iced-Coded** [reply]
It's a somewhat high school, it's normal in our country that some schools do have school emails and some don't.

**Reply by jge162** [reply]
well in high school we don't get school emails. I did not get one at least. you need to show your current report card or something that can verify you. maybe get translated copies too.

**Reply by jge162** [reply]
best of luck to you.

**Reply by Iced-Coded** [reply]
Well, I did show my profile in school diary, where, it shows school, full name (Ф.І.Б (Last, First, Middle name)) and date of birth, it was translated by Google Translate.

**Reply by jge162** [reply]
not sure google translate will work. Needs to be more official than that. talk to someone at the schools admin for help with getting translated official documents. not saying that will work. just an option.

**Reply by Iced-Coded** [reply]
Welp, now I tried sending dated class schedule :)

**Reply by jge162** [reply]
yes keep trying. they will consider and someone will make the decision at github
